dolphinscheduler-data-quality/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,22 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+        
+log4j.rootLogger=INFO, stdout
```

Comment:
I think it is better to use xml files to manage logs, consistent with other modules.

Reply:
Spark use the log4j as the default logger , so i use the log4j.properties. If you want to use logback.xml in spark application , you need to change the actual env lib, otherwise it will not print the log. That is why I use the log4j.properties.

Reply:
I mean we can use log4j.xml :)